cms/djangoapps/contentstore/course_info_model.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=unicode-format-string
```

Comment:
As discussed in other channels, I don't think we should ignore entire files... instead just don't make any changes to them.